tests/test_analysis/test_atomicfluct.py

```diff
@@ -40,6 +40,16 @@ def test_RMSF(self):
         data = pt.rmsf(t0)
         aa_eq(data, state.data[-1].values)
 
+    def test_RMSF(self):
```

Comment:
@envirodug you need to update the method name. There is already a 'test_RMSF'.

may be 'test_RMSD_with_options'?